sample_3.py

```diff
@@ -0,0 +1,5 @@
+class Solution:
+    @lru_cache
```

Comment:
don't know why this is written, no need of it i guess

Reply:
lru_cache decorater is intended for using internal caching for leastly used data, but it is environment dependent. Removing it for general use.